engine/paymail/paymail_service_client.go

```diff
@@ -73,6 +74,35 @@ func (s *service) GetCapabilities(ctx context.Context, domain string) (*paymail.
 	return &response.CapabilitiesPayload, nil
 }
 
+// GetP2PDestinations will ask a paymail host on given address for P2P destinations.
+func (s *service) GetP2PDestinations(ctx context.Context, address *paymail.SanitisedPaymail, satoshis uint) (*paymail.PaymentDestinationPayload, error) {
```

Comment:
In our project - I can see - that the variables types for `satoshis` are inconsistent. (uint, uint64)
I know that on 64-but machines there is no difference between them. 
But... 
Since we are now planning to fix things in spv-wallet - I think we could establish ONE type for these.

Reply:
I'm using now bsv.Satoshis from models package